internal/cmd/login/command.go

```diff
@@ -167,7 +167,10 @@ func (c *command) loginCCloud(cmd *cobra.Command, url string) error {
 		c.printRemainingFreeCredit(cmd, client, currentOrg)
 	}
 
-	return c.saveLoginToKeychain(cmd, true, url, credentials, save)
+	if save && runtime.GOOS == "darwin" && !c.cfg.IsTest {
```

Comment:
Is there any way we can mock the keychain for tests, so we don't need this third condition?

Reply:
unluckily i haven't found it